index.html

```diff
@@ -49,7 +49,7 @@
     </div>
     <!-- header starts -->
     <header class="fix">
-        <video src="video/video.mp4" class="vid-bg" autoplay loop muted></video>
+        <img src="video/gif1.gif" class="vid-bg" alt="background-gif">
```

Comment:
rename the file to `background.gif` so it is more meaningful.

Reply:
I have committed the changes requested. Please see to it :3